src/Kunstmaan/TranslatorBundle/Command/ImportTranslationsFromFileCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Kunstmaan\TranslatorBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Exception\LogicException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CalculateCoordinatesCommand.
+ */
+class ImportTranslationsFromFileCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('kuma:translator:import-file')
+            ->setDescription('Export file with translations, supported formats are xlsx, ods, csv')
+            ->addArgument('file', InputArgument::REQUIRED, 'The full path of the file')
+            ->addOption('force', 'f', InputOption::VALUE_NONE, 'Force import, overwrite all existing database entries');
+    }
+
+    /**
+     * @throws LogicException
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     *
+     * @return int|null|void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $file = $input->getArgument('file');
+        $force = $input->getOption('force');
+
+        $locales = explode('|', $this->getContainer()->getParameter('requiredlocales'));
+        $imported = $this->getContainer()->get('kunstmaan_translator.service.importer.importer')->importFromSpreadsheet($file, $locales, $force);
+        $output->writeln(sprintf("Translation imported: %d", $imported));
+    }
+}
```

Comment:
Can you add a new line at the end of the file? Insight cries if there isn't one 😭